<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFindFromPlot</class>
 <widget class="QDialog" name="DialogFindFromPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Select plot ordinate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QComboBox" name="namesBox"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Abscissa</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Column</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QSpinBox" name="xNum">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="xminVal">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>max</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QDoubleSpinBox" name="xmaxVal">
     <property name="decimals">
      <number>10</number>
     </property>
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ordinate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Column</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QSpinBox" name="yNum">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="yminVal">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>max</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QDoubleSpinBox" name="ymaxVal">
     <property name="decimals">
      <number>10</number>
     </property>
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogFindFromPlot</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogFindFromPlot</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
